### PR TITLE
data: Chintu1509

### PR DESCRIPTION
**Pull request by Chintu1509**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5466"></a>

### CODE REVIEW

File: data/Chintu1509.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "displayStatusPublic":true,
+  "type": "personal",
+  "bio": "Iam a CSE Undergraduate",
+  "avatar":"https://github.com/Chintu1509",
+  "Links":[
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/VengalaUday",
+      "icon": "FaTwitter"
+    }.
```
Comment (vedprakashnautiyal) [inline data/Chintu1509.json:12]: this should be ,

**Review by vedprakashnautiyal — CHANGES_REQUESTED** [review]

File: data/Chintu1509.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Vengala Uday",
+  "displayStatusPublic":true,
+  "type": "personal",
+  "bio": "Iam a CSE Undergraduate",
+  "avatar":"https://github.com/Chintu1509",
```
[the remark this answers is not in the document]
Comment (vedprakashnautiyal) [reply data/Chintu1509.json:6]: remove it from the JSON

**Review by vedprakashnautiyal — CHANGES_REQUESTED** [review]

File: data/Chintu1509.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Vengala Uday",
+  "displayStatusPublic":true,
```
Comment (imbeshat) [inline data/Chintu1509.json:3]:
```suggestion
```

Comment (imbeshat) [reply data/Chintu1509.json:3]: ```displayStatusPublic``` is also not required.

**Comment by SaraJaoude** [comment]
You have conflicts in your file, please resolve. This is usually because your fork is not up to date.  

You may wish to have a look at the following Docs and video: https://linkfree.eddiehub.io/docs/how-to-guides/editing

**Comment by Chintu1509** [comment]
Thanks for your suggestion, I have updated my fork.

**Comment by kumarsonsoff3** [comment]
Duplicate of #5531 

--
Closing this one, as your latest PR with the same changes has been merged recently.

**Pull request closed without merging**